modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -394,7 +394,7 @@ private void generateModel(List<File> files, Map<String, Object> models, String
     }
 
     @SuppressWarnings("unchecked")
-    private void generateModels(List<File> files, List<Object> allModels, List<String> unusedModels) {
+    protected void generateModels(List<File> files, List<Object> allModels, List<String> unusedModels) {
```

Comment:
Same. This should not be protected. If you're doing this for testing, please remove the modifier and treat this as package-private.

Reply:
This update is needed for tests, changed it to package private